src/ncp/rcp_host.cpp

```diff
@@ -322,16 +322,23 @@ void RcpHost::HandleStateChanged(otChangedFlags aFlags)
 
 void RcpHost::Update(MainloopContext &aMainloop)
 {
+    VerifyOrExit(mInstance != nullptr);
```

Comment:
Why we can skip the code when the `mInstance == nullptr`? Shouldn't we add `assert(mInstance != nullptr)`?

Reply:
This is for test purpose. Because the fuzzer tests in Android will call APIs when the host hasn't been initialized. And the unit tests here will also do the same thing. However we need to run the loop to process the tasks and get the async results. And `otTaskletProcess` will be called when `mInstance` is `nullptr`. So we skip here and avoid `otTaskletProcess` from being called in unit tests.

Reply:
I've just come up with a better solution. I removed the `VerifyOrExit(mInstance != nullptr);`.